doc/advanced-configuration/Services.md

```diff
@@ -1,8 +1,25 @@
-### Module Description
-This module significantly extends `mongooseimctl` script capabilities. New functionalities appear as additional commands. For detailed description, just run `mongooseimctl` script with no parameters.
+Some functionalities in MongooseIM are provided by "services".
+A service is similar to a module, only it is started only once with a global configuration, while a module is started for every virtual host and may have global or host-specific configuration.
+Service configuration is similar to a module configuration, e.g.:
+```
+{services, [
+            {service_admin_extra, [{submods, [node, accounts, sessions]}]}
+           ]
+}.
+```
 
-### Options
-* `submods` (default: all submodules): List of function groups added by `mod_admin_extra`. Allowed elements:
+
+## Service list
+
+As of version 2.2, only one module is a "service provider".
+Over time, part of modules which are not host-specific will be refactored to be services.
```

Comment:
Eventually the modules which are not host-specific will be refactored to be services.
?